src/sdp/Utils.cpp

```diff
@@ -390,7 +390,7 @@ namespace mediasoupclient
 			{
 				MSC_TRACE();
 
-				auto jsonMssrcsIt = offerMediaObject.find("ssrc");
```

Comment:
Why this?

Reply:
I refer to project mediasoup-client,
`export function getCname(
	{ offerMediaObject }:
	{ offerMediaObject: any }
): string
{
	const ssrcCnameLine = (offerMediaObject.ssrcs || [])
		.find((line: { attribute: string }) => line.attribute === 'cname');

	if (!ssrcCnameLine)
		return '';

	return ssrcCnameLine.value;
}`

Reply:
ok